src/ManifestStrategy/ConfigureEnvManifestStrategy.php

```diff
@@ -90,6 +90,19 @@ public function apply(array $manifest, string $moduleName, InputOutputInterface
         return true;
     }
 
+    /**
+     * @param string $target
+     *
+     * @return bool
+     */
+    protected function targetExists(string $target): bool
+    {
+        $configFileName = $this->config->getConfigPath();
+        $configFileContent = (string)file_get_contents($configFileName);
```

Comment:
`$configFileName` variable is used only once here, so it is not really needed

Reply:
Missed this after fixes
Fixed